addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -4687,10 +4689,10 @@ int GetMostDamageZom()
 	
 	for (int iClient = 1; iClient <= MaxClients; iClient++)
 		if (IsValidZombie(iClient))
-			if (g_iDamage[iClient] > iHighest) iHighest = g_iDamage[iClient];
+			if (g_iDamageInfected[iClient] > iHighest) iHighest = g_iDamageInfected[iClient];
```

Comment:
Styling can also change to match same as below, single `if` and changing `iHighest` in newline